table/column.go

```diff
@@ -181,7 +181,11 @@ func CastValue(ctx sessionctx.Context, val types.Datum, col *model.ColumnInfo, r
 	if types.ErrOverflow.Equal(err) && returnOverflow {
 		return casted, err
 	}
+<<<<<<< HEAD
```

Comment:
Please fix the conflict @Patrick0308 .

Reply:
Do I need create a new pr to fix this problem?

Reply:
What should I do?